other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,181 @@
+
+## Example launch checklist
+
+```markdown
+# Goal
+
+<!-- A brief overview of the launch objective. Is it a migration? Is it a new site? -->
+
+## Pre-launch checklist
+<!-- To be filled out by the project team/lead. Not needed for Altis Support ticket. -->
+- [ ] Are we deploying the correct branch?
+- [ ] Has AWS SES been setup?
+- [ ] Is a content freeze active?
+- [ ] Is the client ready for DNS switches?
+- [ ] Site is ready for HTTPS?
+- [ ] Are any redirections needed?
+
+## Domains hosted on Altis
+<!-- List of domains that will be on the stack -->
+
+### Production
+- domain.com
+- domain2.com
+- etc.
+
+### Staging
+- domain-staging.altis.cloud
+- domain2-staging.altis.cloud
+- etc.
+
+### Development
+- domain-dev.altis.cloud
+- domain2-dev.altis.cloud
+- etc.
+
+**What domain(s) will email need to be sent from?**
+<!-- Enter as a list -->
+- [ ] noreply@domain.com
+- [ ] etc.
+
+**For each site in the network, what will be the FROM email?
+<!-- Enter as a list -->
+- [ ] **Network Admin:** noreply@domain.com
+- [ ] **domain.com:** some-email@domain.com
+- [ ] etc.
+
+## Environment
+* **GitHub repository:** https://github.com/humanmade/{projectname}
+* **GitHub repository deploy branches:**
+  * **Production:** `main`
+  * **Staging:** `staging`
+  * **Development:** `dev`
+  * **Altis integration branch:** `altis` <!-- If applicable -->
+* [ ] **Database exists in Altis Dashboard?**
+  * [ ] If not, provide a link to the database to import:
+* [ ] **Uploads exist in Altis Dashboard?**
+  * [ ] If not, provide a link to the uploads to import:  
+* **Who is providing the SSL certificate(s) (Altis or client)?:** 
+* **Who controls the DNS for the client (Altis or client)?:**
+* **Special requirements or considerations for ElasticSearch:**
+* **Projected traffic:** 
+* **Specific cookies that need to be allow-listed outside these patterns `comment_`, `hm_`, `wordpress_`, `wp-`, `wp_`, `altis_`:**
+* **Any other specifications or challenges?**
+
+## Altis Cloud
+- [ ] Nameserver records have been provided to the client (if we're managing their DNS).
+- [ ] AWS SES domain verification DNS records have been provided to the client or entered into Route 53.
+- [ ] AWS SES Sending domains have been verified in SES.
+- [ ] AWS SES Sending domains have been configured in the application code.
+- [ ] ACM Certificate DNS records have been provided to the client or entered into Route 53
+- [ ] ACM Certificate has been issued.
+- [ ] ACM Certificate has been attached to the environment.
+- [ ] Existing database, if any, has been imported into the environment.
+- [ ] Existing uploads, if any, have been imported into the environment.
+- [ ] Database has been search/replaced with the environment-specific domain as the replacement.
+
+<!-- For each environment that needs to be created or migrated, include a checklist like the following: -->
+
+### domain-dev
```

Comment:
These sections should just be internal docs replaced by a note to say that once the zendesk ticket has been created a cloud engineer will respond to that ticket with the following information:

specific info depending on some of the above, eg. are we managing their DNS

Reply:
Added a new section between submitting a support ticket and launch that sort of covers this a bit ("the Altis Support team will work on setting up your environments and be in touch with your team"). Let me know if you think anything additional is needed here.